app/presenters/summary/sections/income_benefits_details.rb

```diff
@@ -0,0 +1,70 @@
+module Summary
+  module Sections
+    class IncomeBenefitsDetails < Sections::BaseSection
```

Comment:
As income benefits and payments have the same code, it would be good if they both inherited from the same class. The `income_benefits` and `income_payments` methods could just be called payments and they could be defined in the sub classes